src/microbe_stage/MicrobeStage.cs

```diff
@@ -257,6 +257,8 @@ public void SetupStage()
 
         StartMusic();
 
+        CheatManager.OnCheatsDisabled();
```

Comment:
This shouldn't be here, otherwise all stages need this separately, and also the editor. Instead the main menu code as well as the save load helper should have this instead.